pkg/service/record/record.go

```diff
@@ -178,6 +178,10 @@ func (r *Recorder) Start(ctx context.Context) error {
 				r.telemetry.RecordedTestCaseMock(mock.GetKind())
 			}
 		}
+		err := pkg.CreateGitIgnore(r.logger, r.config.Path)
```

Comment:
Why do you need to create the gitignore file during the record mode? We only need to create it when reports are created.

Reply:
Apologies for the confusion with your previous comment 
> Why is this only in the record mode? What if the user starts the test mode directly by importing the testcases from somewhere?Also we need to check if the testreport was generated successfully, and only then add it to the gitignore file

Removed it from the record command